BlueprintUICommonControls/Sources/ScrollView.swift

```diff
@@ -264,6 +265,7 @@ fileprivate final class ScrollerWrapperView: UIView {
         }
 
 
+        self.scrollView.keyboardDismissMode = scrollView.keyboardDismissMode
```

Comment:
@timdonnelly: Do you remember why all the applications in this method do:

```
if self.scrollView.property != scrollView.property {
   self.scrollView.property = scrollView.property
}
```

?